core/imageroot/var/lib/nethserver/cluster/actions/remove-module/50update

```diff
@@ -78,7 +73,7 @@ for kdef in rdb.scan_iter('*/default_instance/*'):
         rdb.delete(kdef)
 
 # Erase the module keyspace
-module_keys = list(rdb.scan_iter(f'module/{module_id}*'))
+module_keys = list(rdb.scan_iter(f'module/{module_id}/*'))
```

Comment:
No, absolutely: see the PR description.

Reply:
yes... 

it is also possible to add the key to `module_keys` separately, however that naming pattern is weak and I prefer to avoid it also in the future. There are no keys following this bad pattern.

Reply:
Ok, fine